proto/agent.proto

```diff
@@ -3,10 +3,17 @@ syntax = "proto3";
 package proto;
 
 enum ServiceRole {
-  ALL = 0;
-  META = 1;
-  GRAPH = 2;
-  STORAGE = 3;
+  UNKNOWN_ROLE = 0;
+  ALL = 1;
```

Comment:
Maybe we should delete the `all` here:
1. [invoker](https://github.com/vesoft-inc/nebula-br/pull/28/files#diff-f21f0a64a7bcaaa98158775bed752356bcec028af5b1968a73515e717301c158R130) does not use it.
2. If invoker use `ALL` later, IIRC, if `all` is passed to the scripts, the status could be wrong because we compare by `strings.Contains(outStr, "Exit")`. (1 service is dead, the other is alive... etc)

Reply:
Good point! `ALL`  was introduced before for stopping all service at one RPC, but after refactor, it is not used now. I'll remove it